docs/adrs/add-auto-import-embedded-registry.md

```diff
@@ -0,0 +1,21 @@
+# Easy way for auto adding images to k3s
+
+Date: 2024-10-2
+
+## Status
+
+Proposed
+
+## Context
+
+Since the feature for embedded registry, the users appeared with a question about having to manually import images, specially in edge environments
+As a result, there is a need for a folder to be created, where every image there will be watched by a controller (a child process that will run when the embedded registry is created) for changes or new images, this new images or new changes will be added to the node registry, meaning that other nodes will have access to the image.
```

Comment:
It'll be much better if we could reuse the existing folder for images instead of creating  a new one

Reply:
Could possibly be convinced to just change the behavior of the existing agent/images folder.

Reply:
I was thinking into creating the controller for agent/images and the controller will upload the images as the default behavior, but he will finish the process after the first load of images if the user decide to not use embedded registry.